projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-component.scss

```
@@ -544,6 +544,10 @@
         @extend %igx-grid__row-editing-outlet !optional;
     }
 
+    @include e(grid-addrow-snackbar) {
```

The produced selector will be `igx-grid__grid-addrow-snackbar`. This name is rather convoluted.
I would suggest the following change:
```suggestion
    @include e(addrow-snackbar) {
```